xee/ext.py

```diff
@@ -279,9 +279,14 @@ def get_info(self) -> Dict[str, Any]:
       rpcs.append(('projection', self.projection))
 
     if isinstance(self.geometry, ee.Geometry):
-      rpcs.append(('bounds', self.geometry.bounds()))
+      rpcs.append(('bounds', self.geometry.bounds(1, proj=self.projection)))
```

Comment:
If the user specifies an `ee.Geometry`, we should just honor it. No need to transform an existing geometry.

Reply:
My mistake. I thought we were passing the geometry in directly without calling bounds on it.